### DISCUSSION TITLE
Gettting Started

### DISCUSSION THREAD

**Discussion by cmhandan** [Cyfrin/foundry-full-course-cu#2865] [General]
Hey , everyone i am a MERN stack developer , now i am thinking of starting web3 , is this the place where i will not regreting following. And will this course will kepp me engage in the learning process.

**Comment by EngrPips** [comment]
Hello @Mr-mahato, I believe you will do just fine with Patrick's course plus there is a whole community willing to help you whenever you get stuck. Looking forward to seeing you learn.

**Reply by Aashim-Limbu** [reply]
yeah i'm also a MERN stack developer exploring Web 3.0. The course is superb man cannot get this level of resource anywhere .I'm just enjoying my journey as solidity is somewhat similar to class in Javascript so kind of tuning with my view 😁😁. Hope you'll enjoy this.

**Reply by EngrPips** [reply]
Indeed

**Reply by LordAdhaar** [reply]
Indeed
